src/Console/Command/CleanCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Lmc\Steward\Console\Command;
+
+use Lmc\Steward\Console\CommandEvents;
+use Lmc\Steward\Console\Event\BasicConsoleEvent;
+use Lmc\Steward\Publisher\XmlPublisher;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+/**
+ * Clean previous log files in the logs directory
+ */
+class CleanCommand extends Command
+{
+    /** @var Filesystem */
+    private $filesystem;
+
+    /**
+     * @param Filesystem $filesystem
+     */
+    public function setFilesystem(Filesystem $filesystem)
```

Comment:
This function is here mainly because of tests, isn`t it? If so `@internal` annotation should be used.